src/Guardar.js

```diff
@@ -111,6 +111,16 @@ export function has(key) { return keys().includes(key); }
  */
 export function isEmpty() { return size() === 0; }
 
+/**
+ * Get version number
+ * 
+ * @returns {number} number of current version 
+ */
+export function getVersion() {
+	var pjson = require('../package.json');
```

Comment:
I'm using [rollup](https://rollupjs.org/) to bundle the code, and rollup allows using [es modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules).

Probably, you need to do something like this: `import pkg from "../package.json" assert { type: "json" };`

But, I don’t want to load all JSON file just to insert the version, also, a version will always be late, I'm using [np](https://github.com/sindresorhus/np) to publish the package, and it automatically updates the version after build.

So for this version, at least for now, I think it's better to handwrite the version. In version 2 I intend for this to be more automated. Ex: `return "1.0.5";`

Nothing I said is an absolute truth, if you have doubts or don't agree, please comment. : )

Reply:
Totally understandable once you put it that way. 
Not too sure if it will work as I'm still trying to figure the way to actually write the code for the idea that I have. But what if we somehow got the version from the npm registry api json? Using `https://registry.npmjs.com/-/v1/search?text=%3Cguardar%3E&size=1` I get correctly pull up the version number. You think it would be doable to somehow pull it from there?

Reply:
Of course, Thanks for letting me brainstorm on the issue!

Reply:
👌